airflow/www/templates/airflow/dags.html

```diff
@@ -21,11 +21,11 @@
 {% from 'appbuilder/loading_dots.html' import loading_dots %}
 
 {% block page_title %}
-  {% if search_query %}
+  {%- if search_query -%}
```

Comment:
If you are going to do this, do it everywhere, not just in this one page. If you do it to everywhere it should be a separate PR to this one.

Honestly though: this change isn't needed - HTML is not white space sensitive, so this has little benefit.